web/packages/teleport/src/Discover/Database/DeployService/ManualDeploy/ManualDeploy.tsx

```diff
@@ -179,48 +174,51 @@ export function ManualDeploy(props: {
 
   let hint;
   if (showHint && !result) {
-    hint = (
-      <HintBox header="We're still looking for your database service">
+    const details = (
+      <>
         <Text mb={3}>
           There are a couple of possible reasons for why we haven't been able to
           detect your database service.
         </Text>
 
-        <Text mb={1}>
-          - The command was not run on the server you were trying to add.
-        </Text>
-
-        <Text mb={3}>
-          - The Teleport Database Service could not join this Teleport cluster.
-          Check the logs for errors by running{' '}
-          <Mark>journalctl -fu teleport</Mark>.
-        </Text>
+        <ul>
+          <li>
+            <Text mb={1}>
+              The command was not run on the server you were trying to add.
+            </Text>
+          </li>
+          <li>
+            <Text mb={3}>
+              The Teleport Database Service could not join this Teleport
```

Comment:
sorry these aren't your mistakes, but we lowercased them every where else in this file

```suggestion
              The Teleport database service could not join this Teleport
```